linter/allowlists/en.txt

```diff
@@ -88,6 +88,7 @@ Head
 HeaderFooterCode
 helpdesk
 History
+hotspot
```

Comment:
The word `hotspot` has no occurrence in your PR - so you don't need to add this entry to the allow list. Also _hotspot_ is a regular English word and does not need to be put on an allow list for the spell checker.

Reply:
At one point I believe I was using the term "hotspot" and I was getting spellcheck errors. I removed it.